src/utils/viewer/viewer.cpp

```diff
@@ -525,6 +549,106 @@ void Viewer::switchCameraType() {
   }
 }
 
+void Viewer::saveCameraTransformToFile() {
+  if (!Cr::Utility::Directory::exists(saved_transformations_directory)) {
+    Cr::Utility::Directory::mkpath(saved_transformations_directory);
+  }
+
+  // update temporary save
+  lastCameraSave_ = renderCamera_->node().absoluteTransformation();
+  cameraSaved = true;
+
+  // update save in file system
+  saveNodeTransformToFile(renderCamera_->node(),
+                          saved_transformations_directory + "camera." +
+                              getCurrentTimeString() + ".txt");
+}
+
+void Viewer::loadCameraTransformFromFile() {
+  if (!Cr::Utility::Directory::exists(saved_transformations_directory)) {
+    LOG(INFO) << "Cannot find saved transformations directory. Make sure that "
+                 "you've saved a transformation before trying to load one.";
+    return;
+  }
+
+  if (!cameraLoadTimeString_.empty()) {
+    // loading from file system
+    loadNodeTransformFromFile(renderCamera_->node(),
+                              saved_transformations_directory + "camera." +
+                                  cameraLoadTimeString_ + ".txt");
+  } else {
+    // attempting to load from last temporary save
+    LOG(INFO) << "Camera transform file not specified, attempting to load from "
+                 "current instance. Use --camera to specify file to load from.";
+    if (!cameraSaved) {
+      LOG(INFO) << "No transformation saved in current instance. Save a "
+                   "transformation before trying to load one.";
+      return;
+    }
+
+    LOG(INFO) << "Transformation matrix loaded from current instance : "
+              << Eigen::Map<esp::mat4f>(lastCameraSave_.data());
+
+    renderCamera_->node().setTransformation(lastCameraSave_);
+  }
+
+  if (!flyingCameraMode_) {
+    LOG(INFO) << "Note: flying camera mode is OFF, so loading camera transform "
```

Comment:
"Note: the flying camera mode is currently OFF. You may not see any view change."